components/AutocompleteCard.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <v-card class="mb-5">
+    <v-card-title class="mx-2 pb-0">
+      {{ $t('common.filtering.title') }}
+    </v-card-title>
+    <v-col class="mr-2 pt-0 d-flex align-content-start flex-wrap">
+      <v-col v-for="field in distinctfields" :key="field" cols="6">
```

Comment:
```suggestion
      <v-col v-for="(field, key) in distinctfields" :key="key" cols="6">
```

Was getting VueJS errors. This should fix it (`key` needs to be type string/number)